### PR TITLE
feat: show followed by

### PR DESCRIPTION
**Pull request by dzbo**
### Ticket ID

[DEV-11483](https://app.clickup.com/t/2645698/DEV-11483)

### Description

- add "followed by" info 
- fix web3 provider not set issue

CleanShot 2024-08-05 at 10 12 41@2x

### CODE REVIEW

**Comment by Hugoo** [comment]
Task linked: [DEV-11483 Show other profiles that follow the profile you are seeing](https://app.clickup.com/t/2645698/DEV-11483)

File: utils/getPositiveNumber.ts [hunk 1]
```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ *
+ * @param value
+ * @returns
+ */
```
Comment (JeneaVranceanu) [inline utils/getPositiveNumber.ts:6]: I guess we can drop the docs 🙂

Comment (JeneaVranceanu) [reply utils/getPositiveNumber.ts:6]: Or a doc `Returns value if positive or 0 instead.`

**Review by JeneaVranceanu — APPROVED** [review]

File: utils/getPositiveNumber.ts [hunk 2]
```diff
@@ -0,0 +1,18 @@
+/**
+ *
+ *
+ * @param value
+ * @returns
```
Comment (doubleppereira) [inline utils/getPositiveNumber.ts:5]: maybe improve the comment doc here.

File: utils/getPositiveNumber.ts (hunk 1 above)
Comment (dzbo) [reply utils/getPositiveNumber.ts:6]: I will fix the comment

File: utils/getPositiveNumber.ts (hunk 2 above)
Comment (dzbo) [reply utils/getPositiveNumber.ts:5]: I will fix the comment

**Pull request merged**